documentation/staging/content/userguide/managing-domains/model-in-image/auxiliary-images.md

```diff
@@ -339,49 +329,87 @@ Model In Image model files, application archives, and the WDT installation files
+       so that it references the WDT installation directory,
+       which the `imagetool.sh createAuxImage` command
+       defaults to `/auxiliary/weblogic-deploy`
+       (where `/auxiliary` is the default `--wdtHome` value for the `imagetool.sh createAuxImage` command).
+     - `domain.spec.configuration.model.modelHome`
+       so that it matches the `imagetool.sh createAuxImage` command's `--wdtModelHome` value,
+       which defaults to `/auxiliary/models`.
+
+   - Option 2 - Alternatively, you can create the auxiliary image manually by following these steps.
+
+     - First, install WDT in the staging directory and remove its `weblogic-deploy/bin/*.cmd` files, which are not used in UNIX environments:
+       ```shell
+       $ unzip /tmp/mii-sample/model-images/weblogic-deploy.zip -d .
+       $ rm ./weblogic-deploy/bin/*.cmd
+       ```
+       In a later step, we will specify a domain resource `domain.spec.configuration.model.wdtInstallHome`
+         attribute that references this WDT installation directory.
+    
+       If the `weblogic-deploy.zip` file is missing, then you have skipped a step in the prerequisites.
+  
+     - Run the `docker build` command using `/tmp/mii-sample/ai-docker-file/Dockerfile`.
+  
+       ```shell
+       $ docker build -f /tmp/mii-sample/ai-docker-file/Dockerfile \
+         --build-arg AUXILIARY_IMAGE_PATH=/auxiliary \
+         --tag model-in-image:WLS-AI-v1 .
+       ```
+
+       See `./Dockerfile` for an explanation of each build argument.
+
+       {{%expand "Click here to view the Dockerfile." %}}
+       ```
+       # Copyright (c) 2021, Oracle and/or its affiliates.
+       # Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+    
+       # This is a sample Dockerfile for supplying Model in Image model files
+       # and a WDT installation in a small separate auxiliary
+       # image. This is an alternative to supplying the files directly
+       # in the domain resource `domain.spec.image` image.
+    
+       # AUXILIARY_IMAGE_PATH arg:
+       #   Parent location for Model in Image model and WDT installation files.
+       #   Must match domain resource 'domain.spec.auxiliaryImageVolumes.mountPath'
+       #   For model-in-image, the following two domain resource attributes can
+       #   be a directory in the mount path:
+       #     1) 'domain.spec.configuration.model.modelHome'
+       #     2) 'domain.spec.configuration.model.wdtInstallHome'
+       #   Default '/auxiliary'.
+       #
+    
+       FROM busybox
+       ARG AUXILIARY_IMAGE_PATH=/auxiliary
+       ARG USER=oracle
+       ARG USERID=1000
+       ARG GROUP=root
+       ENV AUXILIARY_IMAGE_PATH=${AUXILIARY_IMAGE_PATH}
+       RUN adduser -D -u ${USERID} -G $GROUP $USER
+       COPY ./ ${AUXILIARY_IMAGE_PATH}/
+       RUN chown -R $USER:$GROUP ${AUXILIARY_IMAGE_PATH}
+       USER $USER
+       ```
+       {{% /expand %}}
 
```

Comment:
Consider adding:

1. If you have successfully created the image, then it should now be in your local machine's docker repository. For example:

  ```
  $ docker images model-in-image:WLS-AI-v1
  ```

  ```
  REPOSITORY          TAG                 IMAGE ID            CREATED             SIZE
  model-in-image      WLS-AI-v1           eac9030a1f41        1 minute ago        6.65MB
  ```